core/language/en-GB/Help/rendertiddler.tid

```diff
@@ -1,7 +1,7 @@
 title: $:/language/Help/rendertiddler
 description: Render an individual tiddler as a specified ContentType
 
-(Note: The `--rendertiddler` command is deprecated in favour of the new, more flexible `--render` command)
+<<.deprecated-since "5.1.15" "RenderCommand">>.
```

Comment:
IMO the text is wrong. It's the `--rendertiddler` command that's deprecated and not the "RenderCommand". ... Also the link to the new preferred command is missing now.

I think this problem is valid for all the changes made

Reply:
@pmario the point is that we can't use that macro within core language strings. Therefore if we want to use it we must put it in the non-system, non-shadow wrapper tiddlers we have for these help tiddlers.

Reply:
yea. That's the second problem. .. I just wanted to point out, that the text is wrong too.

Reply:
@pmario which text is wrong?

Reply:
The new text would be wrong too

Reply:
@pmario I thought that the second argument to `deprecated-since` is the title of the tiddler denoting the replacement for the deprecated feature, rather than the deprecated feature's title?

Reply:
> Also the link to the new preferred command is missing now.